src/hotspot/share/opto/mulnode.cpp

```diff
@@ -952,7 +952,8 @@ LShiftNode* LShiftNode::make(Node* in1, Node* in2, BasicType bt) {
 
 //=============================================================================
```

Comment:
While at it, you can also remove this line which we no longer use today
```suggestion
```

Reply:
Done.